core/eolearn/tests/test_core_tasks.py

```diff
@@ -379,47 +379,48 @@ def test_map_features(test_eopatch):
 @pytest.mark.parametrize(
     "feature,  task_input",
     [
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): [2, 4, 8]}),
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): [2]}),
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): (2,)}),
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): 2}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): [2, 4, 6]}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): [2]}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): (2,)}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): 2}),
         (
-            (FeatureType.DATA, "REFERENCE_SCENES"),
+            (FeatureType.DATA, "bands"),
             {(FeatureType.DATA, "B01"): [0], (FeatureType.DATA, "B02"): [1], (FeatureType.DATA, "B02 & B03"): [1, 2]},
         ),
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): []}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): []}),
     ],
 )
 def test_explode_bands(
-    test_eopatch: EOPatch,
-    feature: FeatureType,
+    patch: EOPatch,
+    feature: Tuple[FeatureType, str],
     task_input: Dict[Tuple[FeatureType, str], Union[int, Iterable[int]]],
-):
+) -> None:
     move_bands = ExplodeBandsTask(feature, task_input)
-    patch = move_bands(test_eopatch)
+    patch = move_bands(patch)
     assert all(new_feature in patch for new_feature in task_input)
 
     for new_feature, bands in task_input.items():
         if isinstance(bands, int):
             bands = [bands]
-        assert_equal(patch[new_feature], test_eopatch[feature][..., bands])
+        assert_equal(patch[new_feature], patch[feature][..., bands])
 
 
-def test_extract_bands(test_eopatch):
-    bands = [2, 4, 8]
-    move_bands = ExtractBandsTask((FeatureType.DATA, "REFERENCE_SCENES"), (FeatureType.DATA, "MOVED_BANDS"), bands)
-    patch = move_bands(test_eopatch)
-    assert np.array_equal(patch.data["MOVED_BANDS"], patch.data["REFERENCE_SCENES"][..., bands])
+def test_extract_bands(patch: EOPatch) -> None:
+    bands = [2, 4, 6]
+    move_bands = ExtractBandsTask((FeatureType.DATA, "bands"), (FeatureType.DATA, "MOVED_BANDS"), bands)
+    patch = move_bands(patch)
+    assert np.array_equal(patch.data["MOVED_BANDS"], patch.data["bands"][..., bands])
 
     old_value = patch.data["MOVED_BANDS"][0, 0, 0, 0]
     patch.data["MOVED_BANDS"][0, 0, 0, 0] += 1.0
-    assert patch.data["REFERENCE_SCENES"][0, 0, 0, bands[0]] == old_value
+    assert patch.data["bands"][0, 0, 0, bands[0]] == old_value
     assert old_value + 1.0 == approx(patch.data["MOVED_BANDS"][0, 0, 0, 0])
```

Comment:
this is a weird and unclear test. Figure out what it does and replace it with something better

Reply:
It checks that `patch.data["bands"]` and `patch.data["new"]` are two different object replaced with id test